repos/fdbt-site/src/pages/api/managePassengerTypes.ts

```diff
@@ -93,8 +89,6 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
 
         const nationalOperatorCode = getAndValidateNoc(req, res);
 
-        updateSessionAttribute(req, PASSENGER_TYPE_ATTRIBUTE, singlePassengerType.passengerType);
```

Comment:
Why was this removed?

Reply:
It was unintended functionality - this is setting the users passenger type attribute in their session when they create one. It wasn't a requirement and needed removing.